java/core/src/test/java/sleeper/core/record/process/status/ProcessRunsTest.java

```diff
@@ -190,4 +196,102 @@ public void shouldIgnoreExtraFinishedStatus() {
         assertThat(runs.isFinished()).isTrue();
     }
 
+    @Test
+    void shouldNotCreateProcessRunIfStatusUpdateNotFlaggedAsStartOfRun() {
+        // Given
+        ProcessStatusUpdate notStartedUpdate = () -> Instant.parse("2022-09-24T09:23:30.001Z");
+
+        // When
+        ProcessRuns runs = runsFromUpdates(notStartedUpdate);
+
+        // Then
+        assertThat(runs.getRunList()).isEmpty();
+    }
+
+    @Test
+    void shouldCreateProcessRunFromTwoStartedUpdatesWhenOneIsFlaggedAsStartOfRun() {
```

Comment:
Can we also test the case where we have an update that is the start of the run, followed by an update that is a ProcessRunStartedUpdate but is flagged as not the start of a run?